diff --git a/node_modules/@sentry/hub/dist/scope.js b/node_modules/@sentry/hub/dist/scope.js
index 004166f..e7309bc 100644
--- a/node_modules/@sentry/hub/dist/scope.js
+++ b/node_modules/@sentry/hub/dist/scope.js
@@ -351,7 +351,8 @@ var Scope = /** @class */ (function () {
         this._applyFingerprint(event);
         event.breadcrumbs = tslib_1.__spread((event.breadcrumbs || []), this._breadcrumbs);
         event.breadcrumbs = event.breadcrumbs.length > 0 ? event.breadcrumbs : undefined;
-        event.sdkProcessingMetadata = this._sdkProcessingMetadata;
+        // Merge instead of overwrite to preserve transactionSampling set by @sentry/tracing sample()
+        event.sdkProcessingMetadata = tslib_1.__assign(tslib_1.__assign({}, this._sdkProcessingMetadata), event.sdkProcessingMetadata);
         return this._notifyEventProcessors(tslib_1.__spread(getGlobalEventProcessors(), this._eventProcessors), event, hint);
     };
     /**
